src/components/backend-ai-resource-policy-list.ts

```diff
@@ -819,12 +840,12 @@ export default class BackendAIResourcePolicyList extends BackendAIPage {
   }
 
   /**
-   * Returns human-readable value according to certain conditions
-   *
-   * @param value - raw value
-   * @param enableUnitConvert - if true it enable unit conversion
-   * @return if number then returns number, else if then string
-   */
+  * Returns human-readable value according to certain conditions
+  *
+  * @param {String} value - raw value
+  * @param {Boolean} enableUnitConvert - if true it enable unit conversion
```

Comment:
Shouldn't the first character be lower cases?
Example reference: https://www.typescriptlang.org/docs/handbook/2/functions.html#call-signatures